### DISCUSSION TITLE
Release bundle: strip out .o files before bundling?

### DISCUSSION THREAD

**Discussion by graeme-winter** [dials/dials#1563] [Q&A] [answered]
On Linux the v3.3.1 release bundle appears to contain about 270MB of .o files - which may compress well but are still a lot of extra butes to download and install, and add ~nothing to the average user.

Suggest we should remove these before bundling? But welcome other perspectives.

**Comment by ndevenish** [comment]
Oh, no, do we bundle the full intermediate build? We should absolutely do this. In principle we could also do things like removing the compilers/rebuild the conda environment with only the runtime requirements (would also solve the various readline issues we've seen around).

**Reply by Anthchirp** [reply]
If you go down that route the installer should create a new conda environment and install conda packages into it, ie. sort of where #1198 was supposed to go.

**Comment by Anthchirp** [comment, marked as answer]
Yes, those files should go away. ➡️ #1564
